gapic/src/main/com/google/gapid/models/Capture.java

```diff
@@ -105,10 +106,13 @@ public void loadCapture(File file) {
         return success(capturePath);
       }
     } catch (UnsupportedVersionException e) {
+      handler.reportException(e);
```

Comment:
humm, we might need to decide which side of the RPC fence this should come from.
I think this might already be logged from GAPIS.
This particular example isn't clear either - is this a crash or a analytics event?

Reply:
Good point. Removed.